resources/js/Pages/Home.tsx

```diff
@@ -1,7 +1,48 @@
 import React from "react";
+import { InertiaLink } from "@inertiajs/inertia-react";
+import FormLayout from "@/Atoms/FormLayout";
 
-const Home: React.FC = () => {
-    return <p>This is going to be the Homepage for the Mentorship app.</p>;
+interface Props {
+    heading?: string;
+}
+
+const linkStyles = {
```

Comment:
We can probably just replace these with Tailwind styles directly on the element e.g:

`<span className="underline text-blue-500 px-1">Log out</span>`